test/integration-test/CodeGen/hori_diff_stencil_02_benchmark.cpp

```diff
@@ -39,6 +39,7 @@ TEST(hori_diff_stencil_02, test) {
   cxxnaive::hori_diff_stencil hori_diff_naive(dom, u, out_naive);
 
   hori_diff_gt.run();
+  verif.sync_storages(u);
```

Comment:
again, should be between runs

Reply:
that one was right (probably the only one ;) )